flowtools/metacyto_glm_clr/metacyto_glm_clr.xml

```diff
@@ -0,0 +1,129 @@
+    <exit_code range="15" level="fatal" description="Metadata file must have ONE line per group:filename entity present in cluster stat files." />
+    <exit_code range="16" level="fatal" description="One or more parameters provided are not included in the metadata file." />
+    <exit_code range="17" level="fatal" description="The cluster definition provided wasn't found in the clustering analysis files." />
+    <exit_code range="18" level="fatal" description="The cluster definition provided was found in only one group. Statistical analysis cannot be done." />
+    <exit_code range="19" level="fatal" description="Please provide a valid cluster definition." />
+  </stdio>
+  <command><![CDATA[
+    Rscript --slave --vanilla '$__tool_directory__/metacyto_glm_clr.R' '${metadata}' '${main_var}' '${ma_table}' '${ci}' '${gate}' '${re}' '${loo}'
+  #if $v.more_var=="TRUE"
+    '${v.other_var}'
+  #end if
+  #if $out_plot=="TRUE"
+    'pp' '${plot}'
+  #end if
+  'STAT_FILES'
+  #for f in $input
+    '${f}'
+  #end for
+  ]]>
+  </command>
+  <inputs>
+    <param format="tabular" name="input" type="data_collection" collection_type="list" label="Auto clustering analysis stat output."/>
+    <param format="txt" name="metadata" type="data" label="Metadata file" help="Please see below for formatting."/>
+    <param name="gate" type="text" label="Cluster definition:" value="i.e.:CD3+,CD4-,CD8+,CCR7+"/>
+    <param name="main_var" type="text" label="Variable of interest for GLM" value="i.e.: Gender" help="The variable name needs to be included in the metadata file. By default will be the third column (first and second being group and file names) in the metadata file."/>
+    <param name="ci" type="float" value="0.95" min="0" max="1" label="Confidence Interval" help="The default value is 0.95."/>
+    <conditional name="v">
+      <param name="more_var" type="boolean" label="Add independent variables?" checked="false" truevalue="TRUE" falsevalue="FALSE"/>
+      <when value="TRUE">
+        <param name="other_var" type="text" label="Additional variables of interest for GLM" value="i.e.: Age, Ethnicity" help="Please provide a comma-separated list of variables. The variable names need to be included in the metadata file."/>
+      </when>
+    </conditional>
+    <param name="out_plot" type="boolean" label="Plot result?" checked="false" truevalue="TRUE" falsevalue="FALSE"/>
+  </inputs>
+  <outputs>
+    <data format="tabular" name="ma_table" label="Metadata analysis results of ${input.name}"/>
+    <data format="txt" name="re" label="Random effect model summary statistics for metadata analysis of ${input.name}"/>
+    <data format="tabular" name="loo" label="Leave one out analysis of metadata analysis results of ${input.name}"/>
+    <data format="pdf" name="plot" label="Forest plot of metadata analysis results of ${input.name}">
+      <filter>out_plot</filter>
+    </data>
+  </outputs>
```

Comment:
I would suggest that we add Galaxy tests here, specially if we have example data to use this with. See https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests